mLRS/Common/setup.h

```diff
@@ -407,6 +409,7 @@ void setup_configure(void)
     case SERIAL_BAUDRATE_38400: Config.SerialBaudrate = 38400; break;
     case SERIAL_BAUDRATE_57600: Config.SerialBaudrate = 57600; break;
     case SERIAL_BAUDRATE_115200: Config.SerialBaudrate = 115200; break;
+    case SERIAL_BAUDRATE_230400: Config.SerialBaudrate = 23040; break;
```

Comment:
Is this a typo?  Should it be 230400?

Reply:
oh yeah LOL , sure 230400
I'm btw not sure it's really usefull to go beyond 115200, it's probably more for testing at this point in time